config/application.rb

```diff
@@ -64,6 +64,9 @@ class Application < Rails::Application
     config.email_test_address = ENV["EMAIL_TEST_ADDRESS"]
     config.first_renewal_email_reminder_days = ENV["FIRST_RENEWAL_EMAIL_REMINDER_DAYS"]
 
+    # Renewal
+    config.registration_renewal_grace_window = ENV["REGISTRATION_RENEWAL_GRACE_WINDOW"] || 30
```

Comment:
We merged this fix already, so this is a duplicate now :) https://github.com/DEFRA/waste-exemptions-back-office-ta/blob/3f41bb9b337d45873971d953cdf6a8bdbd5efa68/config/application.rb#L76

Reply:
Oops. A consequence of the rebase! Fixed in deb423c